packages/plugin-lighthouse/src/lib/lighthouse-plugin.integration.test.ts

```diff
@@ -10,3 +10,15 @@ describe('lighthousePlugin', () => {
     expect(pluginConfig.groups).toHaveLength(5);
   });
 });
+
+describe('getRunner', () => {
+  it('should create and execute runner correctly', async () => {
+    // onlyAudits is used to reduce test time
+    const runner = getRunner('https://example.com', {
+      onlyAudits: ['is-on-https'],
+    });
+    await expect(runner(() => void 0)).resolves.toEqual([
+      expect.objectContaining({ slug: 'is-on-https' }),
```

Comment:
Changed report and added numbers. 

BTW, i `used is-on-https` because I thought the test is faster. But it doesn't feel much slower with the new audit...